frontend/src/components/candidates/candidateInterviewsModal.js

```diff
@@ -4,16 +4,28 @@ import React from 'react'
 import { Container, Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
 import InterviewCard from '../interviewCard'
 import InterviewDetails from '../interviewDetails'
+import { getCandidateInterviews } from '../../utils/api'
```

Comment:
instead of passing interviews in as props, pass the ID in and fetch on `componentDidMount`
this makes it easier to update the interviews displayed in the component as its local to the component itself, otherwise I'd have to do redux and I don't really want to do that